py/picca/test/data/config_xcf.ini

```diff
@@ -36,7 +36,7 @@ in tracer2 = SiIII(1207)
 
 [parameters]
 
-bias_QSO  = 109.8 1. None None free
+bias_eta_QSO  = 109.8 1. None None free
```

Comment:
why do you set bias_eta_QSO to such a large number? I guess this is a test that has no impact anywhere else, but I was a bit surprised to see it.